# This message is intended to collect stats about a specific chain in a mulitprompt agent.
# At the time of this comment, only `calls_count` stats are implented.
int32 calls_count # how many times the chain has been invoked since the node startup